### PR TITLE
adding info about DeepPavlov repo

### PR DESCRIPTION
**Pull request by ZotovaElena**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,6 +73,9 @@ Modular file uploader for the browser. Resumable uploads, webcam, screen recordi
   
 - **[DataScript](https://github.com/tonsky/datascript)** by [Nikita Prokopov](https://github.com/tonsky)<br>
   An immutable in-memory database and Datalog query engine in Clojure and ClojureScript.
+  
+- **[DeepPavlov] (https://github.com/deepmipt/DeepPavlov)** by [Fedor Ignatov] (https://github.com/IgnatovFedor)<br>
```
Comment (igoradamenko) [inline README.md:77]: As I see @IgnatovFedor isn't the only author of the repo.

Would you mind to change the author link to “[Neural Networks and Deep Learning lab, MIPT](https://github.com/deepmipt)”?

Comment (igoradamenko) [inline README.md:77]: Oh, and remove the space between ] and (, please.

The project link is broken now.

Comment (ZotovaElena) [reply README.md]: Yes, sure.

Comment (igoradamenko) [reply README.md]: Thanks!

Comment (igoradamenko) [reply README.md]: Ah, relax, I will fix the link by myself.

**Comment by igoradamenko** [comment]
Thanks! 🚀

**Pull request merged**